tests/cloudpickle_test.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 import pickle
 import sys
+import numpy as np
+import scipy.special
```

Comment:
I think we don't want to introduce a strong dependency on numpy & scipy to run the test. I would protect those 2 imports in a `try / except ImportError:` block and raise `nose.SkipTest('missing numpy or scipy')` in the test itself when numpy / scipy are not available.